consensus/src/pipeline/buffer_manager.rs

```diff
@@ -333,6 +339,54 @@ impl BufferManager {
         }
     }
 
+    fn try_add_pending_commit_vote(&mut self, vote: CommitVote) -> bool {
+        let block_id = vote.commit_info().id();
+        let round = vote.commit_info().round();
+        let epoch = vote.commit_info().epoch();
+
+        // Store the commit vote only if it is for one of the next 100 rounds in the same epoch.
+        // As the buffer manager is reset between epoch, we don't benefit from storing the pending commit votes
+        // for the next epoch.
+        if epoch == self.epoch_state.epoch
+            && round > self.highest_committed_round
+            && self.highest_committed_round + 100 > round
+        {
+            if let Some(votes) = self.pending_commit_votes.get_mut(&round) {
+                if votes.len() < 150
```

Comment:
We should check if the voting power is super majority (`check_aggregated_voting_power`), instead of hard-code a magical number.

Reply:
As we are only storing one commit per author in validator_verifier, hardcoding a limit seems unnecessary. Removed the 150 limit.